extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -143,12 +148,15 @@ export class AmpAudio extends AMP.BaseElement {
 
     applyFillContent(audio);
     realChildNodes(this.element).forEach((child) => {
+      if (child.tagName === 'AUDIO') {
```

Comment:
Nit: Could this be
```suggestion
      if (child === audio) {
```

Reply:
Thanks, @jridgewell I have addressed the changes.